boot/freeldr/freeldr/ntldr/winldr.h

```diff
@@ -106,7 +106,8 @@ PVOID WinLdrLoadModule(PCSTR ModuleName, PULONG Size,
 
 // wlmemory.c
 BOOLEAN
-WinLdrSetupMemoryLayout(IN OUT PLOADER_PARAMETER_BLOCK LoaderBlock);
+WinLdrSetupMemoryLayout(IN OUT PLOADER_PARAMETER_BLOCK LoaderBlock,
+                        IN PLOADER_PARAMETER_EXTENSION Extension);
```

Comment:
This change is pretty pointless, since you can get the extension from the LoaderBlock in this function, instead of getting the pointer in the caller and then passing it to the function.

Reply:
it's because the `LoaderBlock->Extension` have the Virtual Address and not the Physical Address.